code-an-off-swtich.md

```diff
@@ -0,0 +1,26 @@
++++
+principle = "Code an off-switch"
+tags = ["feature flag", "feature toggle", "off switch"]
+summary = "Always plan for a way to switch your work off"
+authors = ["Kealan Parr"]
+contributors = ["Kealan Parr"]
```

Comment:
You should remove yourself as a contributor, as you cannot be both a contributor and author

https://principles.dev/documentation/#contributors

Reply:
```suggestion
contributors = []
```